### PR TITLE
fix clim setter

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR fixes a bug where the clim slider wasn't being updated on loading

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)

# How has this been tested?

- [x] run examples

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: napari/_qt/qt_image_layer.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class QtImageControls(QtLayerControls):
     def __init__(self, layer):
         super().__init__(layer)
 
-        self.layer.events.clim.connect(self.clim_slider_update)
+        self.layer.events.clim.connect(lambda e: self.clim_slider_update())
```
Comment (jni) [inline napari/_qt/qt_image_layer.py:13]: Can you explain why this is needed?

Comment (sofroniewn) [reply napari/_qt/qt_image_layer.py:13]: I dropped the 2nd `event` argument for `clim_slider_update` as it wasn't used, but the callback expects it. we use this pattern frequently in the code base

Comment (jni) [reply napari/_qt/qt_image_layer.py:13]: Maybe add "event=None" to the signature instead?

Comment (sofroniewn) [reply napari/_qt/qt_image_layer.py:13]: I prefer having the function call be simpler (i.e. no unnecessary arg). I think @kne42 has advocated for this in the past too. Thoughts @jni?

Comment (jni) [reply napari/_qt/qt_image_layer.py:13]: @sofroniewn I prefer having code that can run without lambdas, *and* unified callback signatures, but I'll leave the discussion for later.

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
@sofroniewn I've approved so you can merge, but I think I would like all of our callbacks to have the same signature, which allows us to write a simple guide for registering callbacks.

File: napari/_qt/qt_image_layer.py (hunk 1 above)
Comment (sofroniewn) [reply napari/_qt/qt_image_layer.py:13]: ok that's fine, we'll standardize in a follow up PR

**Pull request merged**